<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleAnalyse</class>
 <widget class="QDialog" name="SingleAnalyse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
	border: hide;
}

QDialog {
  background-color: rgb(255, 255, 255);
}

QListWidget {
	font: 25 9pt &quot;Microsoft YaHei Light&quot;;
	border: 15px solid white; /* 设置边框的大小，样式，颜色 */
	border-radius: 10px;
}

QListWidget::item {
    height: 30px;
}

QListWidget::item:hover {
	background-color: transparent;
	padding: 10px;
	border-left: 3px solid rgb(130, 130, 130);
}
QListWidget::item:selected {
	background-color: transparent;
	color: black;
	padding: 10px;
	border-left: 3px solid black;
}
</string>
  </property>
  <widget class="QPushButton" name="button_showMainWin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei UI Light</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
  border-radius: 15px;
  border: none;
  background-color: rgb(255, 255, 255);
  text-align: left;
}</string>
   </property>
   <property name="text">
    <string>&lt; 返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>531</width>
     <height>412</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>单独分析</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_1">
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI Light</family>
           </font>
          </property>
          <property name="text">
           <string>选择已录入的简历</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="button_analyse">
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI Light</family>
           </font>
          </property>
          <property name="text">
           <string>分析</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>年龄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="age">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>最高学历</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="education">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>毕业院校</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="college">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>工作年限</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="work">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>推荐岗位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="recommend">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
